CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Changelog
 
+## Unreleased
```

Comment:
nit: don't forget to rebase with `main` after merging

Reply:
of course! 👍